catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -155,8 +168,18 @@ public void configure() throws Exception {
                 }
                 LOGGER.debug("inbox = {}", inbox);
 
-                from(inbox).process(systemSubjectBinder)
-                        .to("content:framework");
+                RouteDefinition routeDefinition = from(inbox);
+
+                if (parameters != null && parameters.size() > 0) {
+                    for (String parameter : parameters) {
```

Comment:
Could do this with streams, but it's not that much better:
```
parameters.stream()
    .map(parameter -> parameter.split("="))
    .filter(paramArr -> paramArr.length == 2)
    .forEach(paramPair -> {
        /* logic */
    });
```

Either way, you don't need the size check on `parameters`.